.github/workflows/CI.yml

```diff
@@ -120,4 +120,4 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.RELEASE_TOKEN }}
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
-        run: npx semantic-release --dry-run
+        run: npx semantic-release
```

Comment:
This is probably all good, but might be sped up by doing a --dev install and using https://github.com/actions/cache to cache node_modules? Not a blocker!

Reply:
yeah that's a good point. i need to update the CI a bit, i'll include in there